src/autoyast-rnc/partitioning.rnc

```diff
@@ -129,6 +132,9 @@ device_order = element device_order {
     element device { text }*
 }
 
+cache_mode = element cache_mode { text }
```

Comment:
maybe we  can have here enum instead of generic string as cache mode is limited?